doc/ros_api.md

```diff
@@ -73,6 +73,17 @@ Note: this topic is only namespaced if a namespace is configured *and* `namespac
 
 ## Services
 
+### inform_job/list
+
+Type: [motoros2_interfaces/srv/ListJobs](https://github.com/gavanderhoorn/motoros2_interfaces/blob/a7cac24d5a22ff756aeb3f3a53ea9cb2258c3e5b/srv/ListJobs.srv)
```

Comment:
note: this points to my fork for now.

Reply:
Just curious why it's on your fork instead of a pending branch of y-global?

Reply:
No particular reason.

Seeing as we should merge a PR to `motoros2_interfaces` adding this definition before this PR anyway, it shouldn't really matter, as I'd update this PR with the correct location.